### PR TITLE
Show data for test files which do not end in html or htm.

### LINKED ISSUE #147: Results not shown for files with certain extensions.
See e.g. http://wpt.fyi/css/CSS2/abspos/abspos-containing-block-initial-001.xht

This page is totally blank. There's no results, and no link to the test on w3c-test.org. If you go up to the containing directory, you'll find results for this test.

### PR DESCRIPTION
**Pull request by AutomatedTester**
This fixes #147.

### CODE REVIEW

File: components/wpt-results.html [hunk 1]
```diff
@@ -219,7 +219,8 @@
       }
 
       _computePathIsATestFile (path) {
-        return path.endsWith('.html') || path.endsWith('.htm')
+        const filePath = /\w+\.\w+/
```
Comment (jeffcarp) [inline components/wpt-results.html]: This won't work, unfortunately there are a few test directories with dots in them:

```
% find . -type d -name "*\.*"
.
./html/dom/documents/dom-tree-accessors/document.getElementsByName
./webgl/conformance-1.0.3
./webgl/conformance-1.0.3/conformance/ogles/GL2Tests/glGetShaderInfoLog_2.0
./webgl/conformance-1.0.3/conformance/ogles/GL2Tests/glGetProgramiv_2.0
./webgl/conformance-1.0.3/conformance/ogles/GL2Tests/glGetProgramInfoLog_2.0
./tools/pytest/extra/setup-py.test
./tools/pytest/_pytest/vendored_packages/pluggy-0.3.1.dist-info
./tools/pytest/.github
./tools/wptrunner/wptrunner.egg-info
./tools/.hypothesis
./css/CSS2/csswg-issues/submitted/css2.1
./css/css-fonts-3/support/fonts/gsubtest-lookup3.ufo
```

Comment (jeffcarp) [reply components/wpt-results.html]: A whitelist seems like the best solution for now until we can cross-reference the manifest in the browser (part of #159). We can add the ones @gsnedders mentioned:

```
gsnedders> looking at sourcefile.py, we need at least html, htm, xhtml, xht, xml, svg, py
```

**Comment by AutomatedTester** [comment]
Updated with a white list as requested

File: components/wpt-results.html [hunk 2]
```diff
@@ -219,7 +219,10 @@
       }
 
       _computePathIsATestFile (path) {
-        return path.endsWith('.html') || path.endsWith('.htm')
+        return path.endsWith('.html') || path.endsWith('.htm') ||
+               path.endsWith('.py') || path.endsWith('.svg') ||
+               path.endsWith('.txt') || path.endsWith('.xhtml') ||
```
Comment (gsnedders) [inline components/wpt-results.html]: `.txt` shouldn't be here

File: components/wpt-results.html [hunk 3]
```diff
@@ -219,7 +219,10 @@
       }
 
       _computePathIsATestFile (path) {
-        return path.endsWith('.html') || path.endsWith('.htm')
+        return path.endsWith('.html') || path.endsWith('.htm') ||
+               path.endsWith('.py') || path.endsWith('.svg') ||
+               path.endsWith('.xhtml') || path.endsWith('.xht') ||
+               path.endsWith('.xml');
```
Comment (jeffcarp) [inline components/wpt-results.html]: Thanks for the change, looks like the JS lint is failing on the `;` (we're using https://standardjs.com/)

**Comment by jeffcarp** [comment]
This is deployed: https://wpt.fyi/css/css-backgrounds-3/border-bottom-left-radius-001.xht

**Pull request merged**